modules/configuring-default-seccomp-profile.adoc

```diff
@@ -1,15 +1,99 @@
 [id="configuring-default-seccomp-profile_{context}"]
-= Configuring the default seccomp profile
-OpenShift ships with a default seccomp profile that is referenced as `runtime/default`. You can enable the default seccomp profile for a pod or container workload by setting `RuntimeDefault` as following:
+= Enabling the default seccomp profile for all {product-title} pods
+{product-title} ships with a default seccomp profile that is referenced as `runtime/default`. In {product-version} newly created pods have the Security Context Constraint (SCC) set to `restricted-v2` and the default seccomp profile applies to the pod. 
 
-.Example
 
-[source, yaml]
+. You can verify Security Context Constraint (SCC) set on a pod by running the following commands: 
+
+.. Verify what pods are running in the namespace: 
++
+[source, terminal]
+----
+$ oc get pods -n <namespace> 
+---- 
++
+For example to verify what pods are running in the `workshop` namespace run the following: 
++
+[source, terminal]
+----
+$ oc get pods -n workshop 
+---- 
++
+.Example output 
++
+[source, terminal]
+----
+NAME                READY   STATUS      RESTARTS   AGE
+parksmap-1-4xkwf    1/1     Running     0          2m17s
+parksmap-1-deploy   0/1     Completed   0          2m22s
+---- 
++
+.. Inspect the pods: 
++
+[source, terminal]
+----
+$ oc get pod parksmap-1-4xkwf -n workshop -o yaml
+----
++
+.Example output 
++
+[source, terminal]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/network-status: |-
+      [{
+          "name": "openshift-sdn",
+          "interface": "eth0",
+          "ips": [
+              "10.131.0.18"
+          ],
+          "default": true,
+          "dns": {}
+      }]
+    k8s.v1.cni.cncf.io/networks-status: |-
+      [{
+          "name": "openshift-sdn",
+          "interface": "eth0",
+          "ips": [
+              "10.131.0.18"
+          ],
+          "default": true,
+          "dns": {}
+      }]
+    openshift.io/deployment-config.latest-version: "1"
+    openshift.io/deployment-config.name: parksmap
+    openshift.io/deployment.name: parksmap-1
+    openshift.io/generated-by: OpenShiftWebConsole
+    openshift.io/scc: restricted-v2 <1>
+    seccomp.security.alpha.kubernetes.io/pod: runtime/default <2>
```

Comment:
Instead of showing this, it may be better to show the pod's/containers securityContext information where the seccomp is configured. These annotations are deprecated iirc.

Reply:
@mvazquezc Here I am simply trying to illustrate that this new restricted-v2 is applied. I added the apps per this guidance https://redhat-scholars.github.io/openshift-starter-guides/rhs-openshift-starter-guides/parksmap-container-image.html#deploy_your_first_image like a test pod really. So maybe these annotations are not deprecated this is the output I captured.

Reply:
The SCC annotation is valid indeed. I was referring to the `seccomp.security.alpha.kubernetes.io/pod` annotation. I'm okay leaving this at it is.